### ISSUE TITLE
HueShift option

### ISSUE THREAD

**Issue by ibillingsley** [Futuremappermydud/ReeSabers.CustomColors#1]
Would be nice to have an option or alternate build using HueShift instead of HueOverride.

Obviously the base saber needs to be red to get the correct color, but it allows for color gradients instead of just solid colors.

Anyway I compiled the mod with this change for my personal use and it seems to work fine.

**Comment by Futuremappermydud** [comment]
I will make it so you can add a file named hueshift inside UserData to toggle it. Is there anything that needs to be done to the preset or how the color is applied?

**Comment by ibillingsley** [comment]
I don't think so. As long as your preset color is mostly red, it ends up matching the note color, even seems to work for greys.
All I did was change `ColorTransformType.HueOverride` to `ColorTransformType.HueShift`
